src/IntroScreen.h

```diff
@@ -22,7 +22,7 @@ class IntroScreen : public Screen
     std::uint16_t               _selected = 0;
     TextList                    _menuItems;
     
-    std::shared_ptr<sf::Sprite>  _sprite;
+    std::shared_ptr<sf::Sprite>  _sprite[8];
```

Comment:
Use vector here too:
```
std::vector<std::shared_ptr<sf::Sprite>> _sprites;
```

Then you can add new default elements to it like so:
```
_sprites->emplace_back()
```

And you can access it like:
```
_sprite.back()->setPosition(0, 0);
```


Just to show you the neatness of this, say you had an object you constructed with a string:
```
struct Foo
{
    Foo(const std::string& id);
    ...
}
```

And then had a vector of `Foo`s, you could do: `myvector.emplace_back("id1");`

What's going to happen is that `emplace_back` passes the argument to the constructor of `Foo` which creates the object "inside" the vector's space. 

It'll work too with `std::shared_ptr` since `emplace_back` will pass the argument to the `std::shard_ptr` constructor, which will then pass it to `Foo`. It's a pretty cool efficiency.

What you have above will do another copy, which is fine in this case since `std::share_ptr` objects are pretty small, but it's good practice to use `emplace_back` (or `emplace_front`) when it's available in whatever container you're using.